docs/querying/sql-functions.md

```diff
@@ -1154,11 +1154,29 @@ Returns the following:
 
 ## GREATEST
 
-`GREATEST([expr1, ...])`
+Returns the maximum value from the provided expressions. The [following rules](sql-scalar.md#reduction-functions) determine how Druid compares between different data types.
```

Comment:
```suggestion
Returns the maximum value from the provided expressions. For information on how Druid interprets the arguments passed into the function, see [Reduction functions](sql-scalar.md#reduction-functions).
```
Since these aren't exactly rules, I think it's best to rephrase.